.github/workflows/lock-closed-issues.yml

```diff
@@ -7,6 +7,10 @@ on:
 jobs:
   lock:
     runs-on: ubuntu-latest
+
+    # don't run this action on forks
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
this won't work at all. The cron does not run on a pull request event

Reply:
thanks for the feedback. adjusted the change per https://github.com/orgs/community/discussions/26684